src/Microsoft.TestPlatform.VsTestConsole.TranslationLayer/TestSession.cs

```diff
@@ -462,6 +472,15 @@ public async Task<bool> StopTestSessionAsync()
     /// <inheritdoc/>
     [Obsolete("This API is not final yet and is subject to changes.", false)]
     public async Task<bool> StopTestSessionAsync(ITestSessionEventsHandler eventsHandler)
+    {
+        return await StopTestSessionAsync(options: null, eventsHandler).ConfigureAwait(false);
+    }
+
+    /// <inheritdoc/>
+    [Obsolete("This API is not final yet and is subject to changes.", false)]
+    public async Task<bool> StopTestSessionAsync(
```

Comment:
Isn't it better to preserve the original signature order? (i.e. keep the event handler arg first).

Reply:
Same answer as above should apply.